### PR TITLE
Add installation docs

### PR DESCRIPTION
**Pull request by evansiroky**
Added super basic documentation on how to install and run Commute.

### CODE REVIEW

**Review by ansoncfit — APPROVED** [review]
Looks good, thanks @evansiroky.  Are there required values that need to be entered in settings.yml (e.g. `geocoder`, `s3bucket`?)

File: README.md [hunk 1]
```diff
@@ -10,4 +10,29 @@
...
+
+## Installation
+
+Commute requires the following to run:
+
+* node.js version 6
+* The [yarn](https://yarnpkg.com/en/) package manager
+* A mongodb database
+* API Keys for the [ESRI geocoder](https://developers.arcgis.com/features/geocoding/)
+* Version v2.4.1-9-g3be6daa of [r5](https://github.com/conveyal/r5/)
```
Comment (ansoncfit) [inline README.md:25]: I.e. this link? https://github.com/conveyal/r5/tree/3be6daa139de55582ecdd730ed38bee3fa7954ef

Comment (evansiroky) [reply README.md:25]: I'll send you more details on slack.

**Comment by evansiroky** [comment]
@ansoncfit Yes, these items are all listed in the default yml files in the `configurations/default` folder.

**Pull request merged**